src/main/java/com/github/dockerjava/core/command/BuildImageCmdImpl.java

```diff
@@ -120,12 +118,12 @@ public Boolean getForcerm() {
     }
 
     @Override
-    public boolean isQuiet() {
+    public Boolean isQuiet() {
```

Comment:
AFAIK `is*` can be used only with primitives, after making it Object it should be `get*`( or `has*`)